### PR TITLE
Support alternative traj name in create_rft_ertobs

### PR DESCRIPTION
**Pull request by berland**

### CODE REVIEW

File: src/fmu/tools/rms/create_rft_ertobs.py [hunk 1]
```diff
@@ -41,23 +41,24 @@
...
-    "absolute_error",
+    "loglevel",
+    "project",
     "relative_error",
+    "rft_prefix",
     "rms_name",
-    "ecl_name",
-    "zonename",
-    "gridname",
+    "trajectory_name",
```
Comment (berland) [inline src/fmu/tools/rms/create_rft_ertobs.py:58]: This is the only change in this list, the rest is just alphabetical sorting.

File: tests/rms/test_create_rft_ertobs.py [hunk 2]
```diff
@@ -157,6 +162,19 @@ def test_get_well_coords():
         == np.array([[0, 0, 0, 0], [2000, 0, 0, 2000]])
     ).all()
 
+    # Alternative trajectory name in RMS:
+    assert (
+        create_rft_ertobs.get_well_coords(
+            rms_project_mock, "R-99", trajectory_name="Imported trajectory"
+        )
+        == np.array([[0, 0, 0, 0], [2000, 0, 0, 2000]])
+    ).all()
```
Comment (JiZur) [inline tests/rms/test_create_rft_ertobs.py:171]: Should it not be an error message sent in case assert test is False ?

Comment (berland) [reply tests/rms/test_create_rft_ertobs.py:171]: Did not understand? This is not an error condition, so no error message.

**Review by JiZur — APPROVED** [review]
Seems alright :)

**Pull request merged**